src/test/java/com/javagirls/MangoRoom/mapper/ReservationMapperTest.java

```diff
@@ -0,0 +1,41 @@
+package com.javagirls.MangoRoom.mapper;
+
+import com.javagirls.MangoRoom.dto.ReservationDto;
+import com.javagirls.MangoRoom.entity.Reservation;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+
+import static com.javagirls.MangoRoom.ReservationMock.getBasicReservation;
+
+public class ReservationMapperTest {
+
+    @Test
+    public void shouldMapReservationToReservationDto(){
+
+        ReservationMapper reservationMapper = new ReservationMapper();
+        Reservation reservation = getBasicReservation();
+
+        final ReservationDto reservationDto = reservationMapper.map(reservation, ReservationDto.class);
+
+        assert(reservation.getId().equals(reservationDto.getId()));
+        assert(reservation.getCheckIn().equals(reservationDto.getCheckIn()));
+        assert(reservation.getCheckOut().equals(reservationDto.getCheckOut()));
+    }
+
+    @Test
+    public void shouldMapReservationDtoToReservation(){
+
+        ReservationMapper reservationMapper = new ReservationMapper();
+
+        ReservationDto reservationDto = new ReservationDto();
```

Comment:
Było by fajnie mieć również ReservationDtoMock myślę że to się może nawet przydać w późniejszych testscg